tests/acceptance/settings-profile-test.js

```diff
@@ -99,12 +100,12 @@ test("it allows editing of users image", (assert) => {
 
   andThen(() => {
     fillInFileInput('.image-drop input', { name: fileName, content: droppedImageString });
-    click('.save');
+    settingsProfilePage.userSettingsForm.clickSave();
   });
 
   andThen(() => {
-    assert.equal(find('.alert-success').length, 1);
-    assert.equal(find('.alert-success p').text(), "Profile updated successfully");
+    assert.ok(settingsProfilePage.successAlert.isVisible);
+    assert.equal(settingsProfilePage.successAlert.message, "Profile updated successfully");
     let expectedStyle = `url(${droppedImageString})`;
     assert.equal(removeDoubleQuotes(find('.image-drop').css('background-image')), expectedStyle);
```

Comment:
For this one, I messed round with it, and where's what I got:

```Javascript
// this is what the entire page object looks like:

import {
  create,
  text,
  visitable,
  findElement
} from 'ember-cli-page-object';
import userSettingsForm from './components/user-settings-form';
import removeDoubleQuotes from '../helpers/remove-double-quotes';


export default create({
  visit: visitable('/settings/profile'),

  successAlert: {
    scope: '.alert-success',
    message: text('p'),
  },

  droppedImage: {
    scope: '.image-drop',
    backgroundImageData() {
      let $el = findElement(this);
      let backgroundImageData = $el.css('background-image');
      return removeDoubleQuotes(backgroundImageData);
    }
  },

  userSettingsForm,
});

// and the assertion:
assert.equal(settingsProfilePage.droppedImage.backgroundImageData(), expectedData);
```

I wanted to use a getter, but it gets evaluated too early, somehow, so I'm not sure what's going on there. Lazy evaluation through a function should be good enough, though.